ivy/src/main/scala/sbt/ivyint/CachedResolutionResolveEngine.scala

```diff
@@ -542,14 +556,6 @@ private[sbt] trait CachedResolutionResolveEngine extends ResolveEngine {
       val result = resolveConflicts(sorted.toList, allModules0)
```

Comment:
Unrelated to this PR, but why is this using `allModules0` rather than `allModules2`?